### ISSUE TITLE
Uso em Salões de Assembleias

### ISSUE THREAD

**Issue by regtec** [AntonyCorbett/OnlyM#14]
Serial interessante algo assim: 
O programa ler como subpastas, e listar você mesmo: 
Exemplo: 
C: \ Usuários ******* \ Documents \ OnlyM \ Media \ Sexta 
C: \ Usuários ******* \ Documents \ OnlyM \ Media \ Sábado 
C: \ Usuários ******* \ Documents \ OnlyM \ Media \ Domingo

![image](https://user-images.githubusercontent.com/39920734/40932540-11565884-6805-11e8-8179-411dc77854b4.png)

**Comment by regtec** [comment]
Uma pena. Pois da forma atual, não é prático o uso em Salão de Assembleias.

**Comment by AntonyCorbett** [comment]
Thanks for your feedback. Perhaps I have misunderstood the requirement. Can you explain how the current solution is impractical please? Regards, Antony

**Comment by regtec** [comment]
Sim. A questão que nos Saloes de Assembleias, são muitos operadores. Em um Congresso, por exemplo, podemos ter até 6 operadores diferentes. Se não for algo que seja muito "didático" ou simples, eles com facilidade irão se atrapalhar.
Além disso, há o fato de tem uma quantidade muito grande de mídias, por dia chega a ter mais de 50.

**Comment by AntonyCorbett** [comment]
Building on your idea, how about having an option to use "dated sub-folders". This would allow you to create sub-folders in the specified media folder which are named according to date, e.g. "2018-06-05" for today. When OnlyM is looking for media it uses any it finds in the media folder, _and_ any files it finds in today's folder. This would allow a user to configure several days of an assembly in advance. However, it still lists 50 items for the day (which may be an issue)

**Comment by regtec** [comment]
Não consegui compreender corretamente sua sugestão.
A opção de se escolher o "DIA" funcionaria assim:
1- A equipe permanente prepara todas as pastas dos dias respectivos.
2- O operador do dia, ao abrir o OnlyM, vai escolher apenas o dia que ele quer usar. Não sendo necessário que ele tenha alguma conhecimento das configurações do programa.

Meu contato: https://api.whatsapp.com/send?phone=5521981978535

**Comment by AntonyCorbett** [comment]
@regtec Thanks for your help. I have implemented a "Calendar Folder" feature. It allows you to configure special sub-folders that apply only on a particular date. Please see the Wiki for details.

**Comment by regtec** [comment]
Boa tarde. Acabei de ver! Só não entendi como funciona ainda.
Pode explicar melhor?

**Comment by regtec** [comment]
Desculpe, não havia lido o Wiki. 
Entendi como foi feito. Só vejo um problema:
Como temos 15 eventos no Salão de Assembleias, seria necessário criar 45 pastas duplicadas com as mídias e respectivas datas.
Por este motivo, que entendo que ter uma opção simples de escolher o Evento, seria mais simples pela quantidade de operadores.

Neste caso, seria o caso de apenas criar 3 subpastas, como: Sexta, Sábado e Domingo.

**Comment by AntonyCorbett** [comment]
Thanks - I understand. Let me give it some thought and get back to you later.
Antony
